Verniana-Jules Verne Studies.js

```diff
@@ -31,51 +30,74 @@
 	<http://www.gnu.org/licenses/>.
 */
 
+// eslint-disable-next-line
+function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
 
-function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
-function doWeb(doc, url) { return FW.doWeb(doc, url); }
+function detectWeb(doc, _url) {
+	if (doc.querySelector('ul.volume_toc li')) return 'multiple';
+	if (doc.querySelector('.ref_on_top')) return 'journalArticle';
+	return false;
+}
+
+function match(t, r) {
+	let m = t.match(r);
+	return m ? m[1] : '';
+}
+
+function scrape(doc, url) {
+	let item = new Zotero.Item('journalArticle');
+	item.url = url;
+	item.title = ZU.capitalizeTitle(ZU.trimInternal(text(doc, '#content h1') || '')) || 'Verniana — Jules Verne Studies';
+	item.creators = [ZU.cleanAuthor(text(doc, 'div#content p.author a') || text(doc, 'div#content p.author'), 'author')];
+	item.abstractNote = unescape(text(doc, 'p.abstract') || '');
```

Comment:
The function `unescape` should not be used anymore, cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape . For what do we need that here?

Reply:
I don't know -- took that from the existing logic. Removed.